src/controllers/objecting.entity.name.controller.ts

```diff
@@ -16,6 +22,58 @@ const validators = [
   check(DIVULGE_INFO_FIELD).not().isEmpty().withMessage(ErrorMessages.SELECT_TO_DIVULGE),
 ];
 
+const getExistingPageData = async (session: Session, res: Response, next: NextFunction) => {
+  try {
+    const objection: Objection = await getObjection(session) as Objection;
+    const existingName: string = objection.created_by.fullName;
+    const existingShareIdneity: string = (objection.created_by.shareIdentity === true) ? "yes" : "no";
```

Comment:
Typo in var name

Reply:
I also believe objection.created_by.shareIdentity === true can be simplified to objection.created_by.shareIdentity

Reply:
Could be, but I'm taking account of undefined as well as false being returned the "===" checks that it is boolean and not undefined.

Reply:
neither created_by or share identity are optional fields in an Objection, so surely it is impossible for it to be null here? (Maybe being Node it is possible)
I would of thought if there is no CreatedBy the getObjection call would error on missing those fields.

Reply:
Updating this just to look for truthy or falsy (which covers false and undefined).